NitroxClient/Logger/ClientLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxClient.Logger
+{
+    public class ClientLogger
+    {
+        [Flags]
+        public enum LogLevel
+        {
+            Disabled = 0,
+            InGameMessages = 1,
+            ConsoleMessages = 2,
+            ConsoleDebug = 4
+        }
+        private static LogLevel logLocations = LogLevel.Disabled;
+
+        // Set with combinaition of enum -- setLogLocation(LOG_CONSOLE | LOG_ERRORMESSAGE)
+        public static void SetLogLocation(LogLevel location)
+        {
+            ClientLogger.logLocations = location;
+            Console.WriteLine("Log location set to " + ClientLogger.logLocations);
+        }
+
+        public static void WriteLine(String msg)
+        {
+            if ((logLocations & LogLevel.ConsoleMessages) != 0) // == LogLevel.ConsoleMessage works as well, but is more verbose
+            {
+                Console.WriteLine(msg);
```

Comment:
Let's add the prefix [Nitrox] to anything that is written to the console. (Same below)